DataProcessing/dataHandler.py

```diff
@@ -29,14 +31,16 @@ def __init__(self, frame_id:int,
         self.label = None
         self.unique_crop_name = None
         self.update_hash()
+        self.is_face = None
 
     def set_label(self, label):
         self.label = label
         self.update_hash() # update unique_hash
 
     def update_hash(self):
          # save format - f'video_name:{self.video_name[9:]}_track_id:{self.track_id}_cam_id:{self.cam_id}_frame_id:{self.frame_id}_crop_id:{self.crop_id}_label:{self.label}'
-        self.unique_crop_name = f'{self.label}_v{self.video_name[9:]}_f{self.frame_id}_b{str(self.bbox)}_t{self.track_id}_c{self.cam_id}_cid{self.crop_id}'
+        bbox_as_list = str(list(self.bbox))
+        self.unique_crop_name = f'{self.label}_v{self.video_name[8:]}_f{self.frame_id}_b{bbox_as_list}_t{self.track_id}_c{self.cam_id}_cid{self.crop_id}'
```

Comment:
1. Better to set self.video_name already in the init to the desired name without the "IPCamera_" prefix
2. Same for bbox_as_list - if we don't need it as np.array anywhere else, then let's set it to list already in the init

Reply:
1. regarding the name --- the video might not hold the prefix IPCam... so I do not think we should remove it.

Reply:
1. So it actually strengthens what I'm saying, it is better we set the name of the video once in the init, so in case the name is different (for example without "IPCamera_"), we don't need to search in the code where we rely on the naming convention.

Reply:
1. Ah now i see what you mean. Will fix